modular_skyrat/modules/company_imports/code/armament_datums/nakamura_modsuits.dm

```diff
@@ -0,0 +1,345 @@
+	item_type = /obj/item/mod/module/energy_shield
+	lower_cost = CARGO_CRATE_VALUE * MODULE_HELLA_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_HELLA_UPPER
+	interest_required = HIGH_INTEREST
+	restricted = TRUE
+	interest_addition = COMPANY_INTEREST_GAIN_BIG
+
+// Utility modules, general purpose stuff that really anyone might want
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules
+	subcategory = "MOD Utility Modules"
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/flashlight
+	item_type = /obj/item/mod/module/flashlight
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/regulator
+	item_type = /obj/item/mod/module/thermal_regulator
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/mouthhole
+	item_type = /obj/item/mod/module/mouthhole
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/signlang
+	item_type = /obj/item/mod/module/signlang_radio
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/plasma_stabilizer
+	item_type = /obj/item/mod/module/plasma_stabilizer
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/basic_storage
+	item_type = /obj/item/mod/module/storage
+	lower_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_CHEAP_UPPER
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/expanded_storage
+	item_type = /obj/item/mod/module/storage/large_capacity
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/retract_plates
+	item_type = /obj/item/mod/module/plate_compression
+	lower_cost = CARGO_CRATE_VALUE * MODULE_MID_LOWER
+	upper_cost = CARGO_CRATE_VALUE * MODULE_MID_UPPER
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nakamura_modsuits/utility_modules/magnetic_deploy
```

Comment:
i'll have to give thought to this one, but same reasoning as above
however it does have contractor colors so it may want a subtype